SeleniumPracticalExercise/PageObjects/PimEmployee.cs

```diff
@@ -0,0 +1,35 @@
+﻿using OpenQA.Selenium;
+using SeleniumPracticalExercise.PageObjects.Common;
+
+namespace SeleniumPracticalExercise.PageObjects
+{
+    public class PimEmployee : BasePageLocal
+    {
+        private readonly By PIMLink = By.XPath("//span[text()='PIM']"); // locate PIM button\
+        private readonly By AddBtn = By.CssSelector("div.orangehrm-header-container > button"); // locate Add button
+        private readonly By FirstNameTxtBox = By.Name("firstName");  // locate firstname txtbox
+        private readonly By LastNameTxtBox = By.Name("lastName");    // locate lastname txtbox
+
+
+        public PimEmployee(IWebDriver driver) : base(driver)
+        {
+        }
+
+
+        public void ClickPIM()
+        {
+            Click(PIMLink);
+        }
+
+        public void ClickADD()
```

Comment:
Rather than two methods... one to click Add and one to enter the names... think of it like a single action... adding a new employee. Create a single new method to do this that contains all actions needed to complete this task. See Login().

Reply:
added them together and renamed it to ClickPIMAdd

Reply:
You have ClickPIMAdd() and EnterNames()... what action is the user taking using both of these methods combined? What is the user trying to do?

The user thinks... I need to do X and the steps are:
1. Click PIM
2. Click Add
3. Enter first/last name

What is X? Name your method X.